ui/src/App.js

```diff
@@ -70,8 +70,9 @@ class App extends Component {
         this.setState({
           searchResults: data.search_results.map(searchResult => {
             return {
-              label: searchResult.display_text,
-              value: searchResult.elasticsearch_field_name,
+              facetName: searchResult.facet_name,
```

Comment:
Update the [comments here](https://github.com/DataBiosphere/data-explorer/blob/776cc4667d5df287a6ba8537e7d8f775f2126903/ui/src/App.js#L36).
Doesn't need to be long, just something like:
```
      // Search results shown in the search drop-down.
      // This is an array of dicts, where each dict has facetName, facetDescription, esFieldName, and facetValue
```
By convention, let's document unobvious state variables in the state initialization section.

Reply:
Done